src/workerd/jsg/memory.h

```diff
@@ -536,7 +536,7 @@ void MemoryTracker::trackField(
     decCurrentNodeSize(sizeof(T));
   }
   pushNode(nodeName.orDefault(edgeName), sizeof(T), edgeName);
-  for (const auto item : value) {
+  for (const auto& item : value) {
```

Comment:
What does this change do? Originally were we copying the value and now we don't? So:
* The original code trigger a compile error if T doesn't have a compile error?
* If T was a large value before, we would copy each entry just to calculate it's size?

Reply:
Original was a bug. This template previously wasn't being used yet so it hadn't been caught.